### DISCUSSION TITLE
pic-release/

### DISCUSSION THREAD

**Discussion by giscus** [narwhrl/narwhrl.github.io#5] [Giscus]
# pic-release/

h3{         font-family: 'Noto Sans Serif', serif !important     }      .center {         margin: auto;         width: 50%;         padding: 10px;     }                               SEASON 1

https://startrekcn.cn/pic-release/

**Comment by rrrexfield** [comment]
Picard的s3e1怎么还没上啊

**Reply by narwhrl** [reply]
一般情况都是周日更新，现在已经上啦

**Comment by EnnawYang** [comment]
1080p少了E05

**Reply by EnnawYang** [reply]
S03E05

**Reply by narwhrl** [reply]
感谢提醒 现已补全
